src/main/java/org/spongepowered/api/world/gen/PopulatorObject.java

```diff
@@ -28,7 +28,10 @@
 import org.spongepowered.api.world.World;
 
 /**
- * Represents a structure which can be placed into the world by a populator.
+ * Represents a relatively small object (must fit in a single chunk) that can be
```

Comment:
A Nether Fortress would be considered a PopulatorObject no? If so, that can easily go across chunk boundaries.